src/components/HomepageCallToAction/index.tsx

```diff
@@ -1,22 +1,45 @@
 import React from 'react';
-import Link from '@docusaurus/Link';
-import { Grid, Heading, Paragraph } from '@amsterdam/design-system-react';
+import { Grid, Heading, Image, Overlap, Paragraph } from '@amsterdam/design-system-react';
+import ShareButton from '@site/src/theme/Button';
 
-const HomepageCallToAction = () => (
-  <Grid.Cell span={12}>
-    <Heading size="level-4">Greetings!</Heading>
-    <Paragraph>
-      Welcome to our website. Here, you&apos;ll discover an overview of all the standards and shared
-      components maintained by the municipality of Amsterdam. While the website is still under
-      development, we cordially invite contributions from our developers. Let&apos;s collaborate to
-      enhance this platform together!{' '}
-    </Paragraph>
-    <Paragraph>
-      {' '}
-      Discover how you can contribute on{' '}
-      <Link to="/docs/intro">our page called &quot;Collaborating on standards&quot;.</Link>
-    </Paragraph>
-  </Grid.Cell>
-);
+interface HomepageCallToActionProps {
+  isBallRolling?: boolean;
+  isImageShowing?: boolean;
+}
+const HomepageCallToAction = ({
+  isBallRolling = true,
+  isImageShowing = true,
+}: HomepageCallToActionProps) => {
+  return (
+    <Grid.Cell span={12}>
+      <Overlap>
+        <Grid className="ams-grid_inner">
+          <Grid.Cell span={7}>
+            <Heading size="level-2">
+              {isBallRolling ? "Let's get the ball rollin'" : 'Contribute'}
```

Comment:
This is confusing; I would just set a prop `title` or `heading` but in this case it's unncessary so I would remove.

Reply:
Is removed, now only Contribute is shown